test/config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "sthFetchInterval": "120s",
+  "metricsAddr": ":1971",
+  "logs": [
+    {
+      "uri": "https://ct.cloudflare.com/logs/nimbus2018/",
```

Comment:
I'd rather not use other people's logs as our default configuration example without their permission. Can we use Birch as our example instead?

Reply:
I'm not sure. @rolandshoemaker is Birch publicly accessible? I don't really want to have to jump through hoops to get IPs whitelisted while developing this tool.

Reply:
Yup Birch is publicly available.